### PR TITLE
Eg 12 pokemons cry

### LINKED ISSUE #12: Pokémon cry's
**Is your feature request related to a problem? Please describe.**
When a new Pokémon gets put on screen its cry will play

### PR DESCRIPTION
**Pull request by BlackCubes**
## Changes
1. Created an object of the Pokemon's noises with their names and urls in pokeCries.js.
2. Created a PokemonsCries component that renders the audio of the Pokemon's noises.
3. Passed isPokeball prop to SinglePokemon component as prop in Main.js.
4. Rendered PokemonsCries component in EnemyPokemon.js.
5. Conditionally rendered the PokemonsCries component with isPokeballRendering prop in SinglePokemon.js.
6. Added a reactoad.mp3 sound.
7. Rendered the reactoad.mp3 in useEffect if that pokemon's name is present in PokemonCry.js.
8. Created a useRef call audioRef in PokemonCry.js.
9. Reduced the first useEffect in PokemonCry.js.
10. Expanded the second useEffect to change the audioRef in PokemonCry.js.
11. Increased the setTimeout length in the second useEffect in PokemonCry.js.
12. Removed the autoPlay attribute in audio element in PokemonCry.js.
13. Added the audioRef variable as ref in audio element in PokemonCry.js.
14. Styled the pokemon-cry class in PokemonCry.css.

## Purpose
The problem was that the Pokemons were not generating any noises when they first appeared.

## Approach
This addresses the problem by rendering out the Pokemon's noises when the user selects it. This creates an engaging and better UI experience.

Closes #12

### CODE REVIEW

File: pokemon-app/src/components/PokemonCry.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        // Decrease the audio volume.
+        audioRef.current.volume = 0.15;
+        // Play the audio.
+        audioRef.current.play();
+      }
+
+      // Whether or not the current audio reference exists,
+      // remove the rendered component so it won't just "stay" there.
+      const timer = setTimeout(() => setIsPokemonsCries(false), 3000);
+      return () => clearTimeout(timer);
```
Comment (Lap343) [inline pokemon-app/src/components/PokemonCry.js:46]: love the comments plus the audio play implementation was great

**Review by Lap343 — APPROVED** [review]
Great job working on the audio cries. Code looks good

**Review by JayDampitan — APPROVED** [review]
Elias cant wait to merge it, had to visit us. Looks great brotha. Keep up the good work.

File: pokemon-app/src/components/PokemonCry.js (hunk 1 above)
Comment (Wolftownn) [inline pokemon-app/src/components/PokemonCry.js:46]: So good!

**Review by Wolftownn — APPROVED** [review]
Great job Elias!

**Review by connielion — APPROVED** [review]
Nice comments. Can't wait to hear reactoads cry.

File: pokemon-app/src/components/PokemonCry.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  // Set the sound url to an empty string
+  const [pokeSoundUrl, setPokeSoundUrl] = useState("");
+  // Initialize the audio reference.
+  const audioRef = useRef();
+
+  // First useEffect sets the rendering to true, and sets the sound url
+  // which depends on the pokemon's name.
+  useEffect(() => {
+    // If either names exist, continue.
+    if (pokeName === "Reactoad" || pokeCries[pokeName]) {
```
Comment (dannysanchez559) [inline pokemon-app/src/components/PokemonCry.js:17]: reactoad pokemon? :0

Comment (BlackCubes) [reply pokemon-app/src/components/PokemonCry.js:17]: I love you and miss you 😘 @dannysanchez559

**Review by dannysanchez559 — APPROVED** [review]
well done, glad you guys were able to get these cries figured out

**Pull request merged**